services/net/transcription/TranscriptionManager.cs

```diff
@@ -232,40 +232,50 @@ private async Task UpdateTranscriptionAsync(ContentModel content)
         // Remote storage locations may not be easily accessible by this service.
         var path = content.FileReferences.FirstOrDefault()?.Path;
         var safePath = Path.Join(_options.VolumePath, path.MakeRelativePath());
+       
         if (File.Exists(safePath))
         {
             this.Logger.LogInformation("Transcription requested.  Content ID: {Id}", content.Id);
```

Comment:
This should be moved down to where a request has actually been made.  At this point it may still not be performed.